components/error-summary/src/index.tsx

```diff
@@ -155,8 +156,4 @@ ErrorSummary.propTypes = {
   ),
 };
 
-ErrorSummary.defaultProps = {
-  heading: 'There is a problem',
-};
-
```

Comment:
this was overwriting the `defaultProps` set above - oops!

Reply:
looks like there's no change to behaviour though? So no need to mark this as 'bugfix' or anything?

Reply:
it was essentially a hidden (non-reported) bug - I think the only affect of it was that if no errors were given the component would crash (as the defaulting of the `errors` prop to `[]` was previously being lost)

Reply:
ok have added bug and tests labels to the PR so we know

Reply:
minor - I'm thinking that this should be mentioned in the PR description as it's relevant to release notes

Reply:
updated